fdbrpc/FlowTransport.actor.cpp

```diff
@@ -1065,7 +1082,8 @@ ACTOR static void deliver(TransportData* self,
 			TraceEvent(SevError, "ReceiverError")
 			    .error(e)
 			    .detail("Token", destination.token.toString())
-			    .detail("Peer", destination.getPrimaryAddress());
+			    .detail("Peer", destination.getPrimaryAddress())
```

Comment:
use `PeerAddr`?

Reply:
The motivation of using "Peer" rather than "PeerAddr" is that the original code is using "Peer". To ensure the backward compatibility, it is reasonable to keep both old and new keys for some time and address the duplicated fields in a future release, when the TraceEvent consumers has been updated properly. @johscheuer your ideas about this?

Reply:
From a compatibility perspective it would be nice to have both for one minor release and then remove it in the next release. I'm no sure about the "cost" of this change as we are adding additional data to the trace events, but I would guess that the additional overhead should be okay.